### PR TITLE
Fix for Filtering Feature

### PR DESCRIPTION
**Pull request by mushthaq33**

### CODE REVIEW

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java [hunk 1]
```diff
@@ -1875,8 +1881,15 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                         }
                         fields.put(subParts[0].replace(":", "_"), value);
                     } else {
-                        String value = subParts[1];
-                        if(value.contains(" ")) {
+                        String value = subParts[1].toLowerCase();
+
+                        if(value.contains(" or ")){
+                            String[] values = value.split(" or ");
+                            for(int i=0; i<values.length; i++){
```
Comment (thushara35) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Apparently, there are some code formatting issues in this code block.

Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Thanks for pointing it out. It is fixed with https://github.com/wso2/carbon-governance/pull/239

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java [hunk 2]
```diff
@@ -1866,7 +1866,13 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                 } else {
                     if(subParts[0].contains(":")) {
                         String value = subParts[1].toLowerCase();
-                        if(value.contains(" ")) {
+                        if(value.contains(" or ")){
```
Comment (splinter) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Code Formatting issue

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java [hunk 3]
```diff
@@ -1866,7 +1866,13 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
...
+                            String[] values = value.split(" or ");
+                            for(int i=0; i<values.length; i++){
```
Comment (splinter) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Code Formatting issue

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java [hunk 4]
```diff
@@ -1875,8 +1881,15 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                         }
                         fields.put(subParts[0].replace(":", "_"), value);
                     } else {
-                        String value = subParts[1];
-                        if(value.contains(" ")) {
+                        String value = subParts[1].toLowerCase();
+
+                        if(value.contains(" or ")){
```
Comment (splinter) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Code Formatting issue

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java (hunk 1 above)
Comment (splinter) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Code Formatting issue

Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Thanks for the comment. This issue has been fixed with #239

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java (hunk 2 above)
Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Thanks for the comment. This issue has been fixed with #239

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java (hunk 3 above)
Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Thanks for the comment. This issue has been fixed with #239

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java (hunk 4 above)
Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Thanks for the comment. This issue has been fixed with #239

Comment (sagara-gunathunga) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: The way you use String literals in above method is bit fragile, please use proper constants.

Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Thanks for the comment. This code has been modified by the following PR - https://github.com/wso2/carbon-governance/pull/239

**Pull request merged**